packages/expo/package.json

```diff
@@ -86,6 +86,7 @@
     "expo-keep-awake": "~14.0.1",
     "expo-modules-autolinking": "2.0.2",
     "expo-modules-core": "2.0.6",
+    "react-native-edge-to-edge": "~1.5.0",
```

Comment:
not sure whether we would like to add react-native-edge-to-edge to bundledNativeModules.json. in case people explicitly add the dep with different versions

Reply:
I'm not sure either 🤔 The PR is structured in a way that their version should get autolinked instead of ours so I think it should be fine?

Reply:
i think it's worth adding it, and if people want to override the version they can use our usual approach for it and explicitly ignore the recommended version: https://docs.expo.dev/versions/latest/config/package-json/